presto-hive/src/main/java/com/facebook/presto/hive/parquet/ParquetHiveRecordCursor.java

```diff
@@ -497,17 +505,19 @@ public PrestoParquetRecordReader(PrestoReadSupport readSupport)
         private final boolean useParquetColumnNames;
         private final List<HiveColumnHandle> columns;
         private final List<Converter> converters;
+        private final TypeManager typeManager;
```

Comment:
do you need this as a field? This is not used anywhere, the constructor arg `typeManager` is passed to the `createGroupConverter()` call.